web/src/main/webapp/WEB-INF/views/jsp/index.jsp

```diff
@@ -5,45 +5,92 @@
     <link href="${bootstrapCss}" rel="stylesheet" />
     <link href="${coreCss}" rel="stylesheet" />
-    <!-- jQuery library -->
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
-    <!-- Latest compiled Bootstrap -->
-    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <link href="${jumbotronCss}" rel="stylesheet" />
+    <script src="${jQuery}"></script>
+    <script src="${bootstrap}"></script>
     <title>Robot library checker</title>
 </head>
 <body>
-<div class="logo">
-    <h1>Robot library checker</h1>
-</div>
-<div class="panel">
-    <div id="libraries">
-        <ul>
-            <li>Library</li>
-            <hr>
-            <li>Library</li>
-            <hr>
-            <li>Library</li>
-            <hr>
-            <li>Library</li>
-        </ul>
-
+<nav class="navbar navbar-inverse navbar-fixed-top">
+      <div class="container">
+        <div class="navbar-header">
+          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
+            <span class="sr-only">Toggle navigation</span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+            <span class="icon-bar"></span>
+          </button>
+          <a class="navbar-brand" href="https://github.com/caroo7/Caro-robot">Github Repository</a>
+        </div>
+      </div>
+    </nav>
+<div class"container text-center">
+ <div class="jumbotron">
+      <div class="container">
+        <h1>Robot library application</h1>
+      </div>
     </div>
+<div class="container">
+    <div id="libraries" class="list-group">
+        <a href="#" class="list-group-item active">Empik</a>
+        <a href="#" class="list-group-item active">Library2</a>
+        <a href="#" class="list-group-item active">Library3</a>
+        <a href="#" class="list-group-item active">Library4</a>
+        <a href="#" class="list-group-item active">Library5</a>
+</div>
     <div id="books">
-        Here we will see the list of books
-             <table>
-                <c:forEach var="b" items="${books}">
+
+           <table class="table table-bordered">
+            <thead>
+             <tr id="heads">
+             <th>Title</th>
+             <th>Author</th>
+             <th>Price</th>
+             <th>Discount</th>
+             <th>Description</th>
+             <th>Tags</th>
+             <th>Genre</th>
+             </tr>
+                <c:forEach var="b" items="${books}" varStatus="myIndex">
                 <tr>
                     <td>${b.getTitle()}</td>
                     <td>${b.getAuthors()}</td>
+                    <td>${b.getPrice()}</td>
+                    <td>${b.getDiscount()}</td>
+                    <td>
+                    <script style="text/javascript">
+                               $(function(){
```

Comment:
The object is implemented in a method, in the jsp I am only iterating through a list of objects and getting the fields I want :)